src/styles/app-header/app-header.css

```diff
@@ -1,11 +1,17 @@
 .app-header {
-  background: var(--color-primary);
-  color: var(--color-white);
-  padding: 12px;
+  background: rgb(72, 71, 102);
+  color: rgb(245, 245, 245);
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
a different way to get the header fixed to the top would be to make the body a grid like
```
display: grid;
grid-template-rows: 48px auto;
```
the main section, which would be in the auto space, could get a
`overflow-y: scroll;` to make the content scrollable
i think this would be more elegant and makes things in the future easier
hope this makes sense

Reply:
thanks, i want to try :)